index.html

```diff
@@ -29,17 +29,159 @@
+            <h2 class="project-name">multi-post stories</h2>
+            <div class="meta">
+                <p>canopy</p>
+                <img src="elements/counter.png" alt="">
+                <p>back end dev</p>
+                <img src="elements/counter.png" alt="">
+                <p>2015</p>
+            </div>
+            <p class="second-text">A daily selection of privately personalized reads;
+                   no accounts or sign-ups required.
+            </p>
+            <ul class="tools">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <a href="#" class="button" target="_blank" rel="noopener noreferrer">see project</a>
+        </li>
+        <li class="project">
+            <div class="screenshot">
+                <img src="elements/snapshoot-3.png" alt="snapshoot-1">
+            </div>
+            <h2 class="project-name">tonic</h2>
+            <div class="meta">
+                <p>canopy</p>
+                <img src="elements/counter.png" alt="">
+                <p>back end dev</p>
+                <img src="elements/counter.png" alt="">
+                <p>2015</p>
+            </div>
+            <p class="second-text">A daily selection of privately personalized reads;
+                   no accounts or sign-ups required.
+            </p>
+            <ul class="tools">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <a href="#" class="button" target="_blank" rel="noopener noreferrer">see project</a>
+        </li>
+        <li class="project">
+            <div class="screenshot">
+                <img src="elements/snapshoot-4.png" alt="snapshoot-1">
+            </div>
+            <h2 class="project-name">multi-post stories</h2>
+            <div class="meta">
+                <p>canopy</p>
+                <img src="elements/counter.png" alt="">
+                <p>back end dev</p>
+                <img src="elements/counter.png" alt="">
+                <p>2015</p>
+            </div>
+            <p class="second-text">A daily selection of privately personalized reads;
+                   no accounts or sign-ups required.
+            </p>
+            <ul class="tools">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <a href="#" class="button" target="_blank" rel="noopener noreferrer">see project</a>
+        </li>        
+    </ul>
+
+    <section class="about">
+        <div class="wrapper">
+            <h2 class="primary-text">About Myself</h2>
+            <p class="secondary-text">Im a software!, I can help you build a product, feature or website look through some of my work and experience! 
+                If you like what you see and have a project you need coded,don't hesitate to contact me.</p>
+            <h2 class="third-text">let's connect</h2>
```

Comment:
- [ ] **OPTIONAL** Good job making this a `<hx>` tag as requested by the previous reviewer. However, you have the choice of improving the Semantic HTML for this element by adding a `<a>` tag for it since it's a link.

> I'm sorry this didn't show up for you earlier.